Public/Src/Tools/SandboxedProcessExecutor/Program.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using BuildXL.Processes;
+using BuildXL.ToolSupport;
+
+namespace BuildXL.SandboxedProcessExecutor
+{
+    internal sealed class Program : ToolProgram<Args>

```

Comment:
Looking at it, seems to me like we should merge this with SandboxExec at some point and have only one tool.

Reply:
Agree. At the moment I'm a bit hesitant to change SandboxExec because things would look ugly. I created AB#1516956 to track the issue.


---
In reply to: [277769688](https://github.com/Microsoft/BuildXL/pull/159#discussion_r277769688) [](ancestors = 277769688)